lib/temporal/testing/temporal_override.rb

```diff
@@ -64,7 +64,7 @@ def start_locally(workflow, *input, **args)
         run_id = SecureRandom.uuid
 
         if !allowed?(workflow_id, reuse_policy)
-          raise TemporalThrift::WorkflowExecutionAlreadyStartedError,
+          raise Temporal::Api::ErrorDetails::V1::WorkflowExecutionAlreadyStartedFailure,
```

Comment:
This is no longer an exception class so not sure what you'd like to raise instead here.

Reply:
We'll have to define all these errors in the clients and then adopt Cadence to raise the same ones. This way we'll have a strong separation between the API and the core logic